gneiss/balances.py

```diff
@@ -0,0 +1,115 @@
+from __future__ import division
+import numpy as np
+from skbio.stats.composition import clr_inv
+from collections import OrderedDict
+
+
+def _balance_basis(tree_node):
+    """ Helper method for calculating balance basis
+    """
+
+    n_tips = sum([n.is_tip() for n in tree_node.traverse()])
+    counts = _count_matrix(tree_node)
+    counts = OrderedDict([(x, counts[x])
```

Comment:
Now, wondering if is worth having this logic within _count_matrix, what do you think?